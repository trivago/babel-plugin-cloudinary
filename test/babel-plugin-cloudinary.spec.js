import * as babel from "babel-core";
import plugin from "../lib";

describe("babel-plugin-cloudinary", () => {
  it("sample", () => {
    const input = "const a = url(1)";
    const {
      code,
    } = babel.transform(input, {
      plugins: [plugin],
    });

    expect(code).toMatchSnapshot();
  });
});
