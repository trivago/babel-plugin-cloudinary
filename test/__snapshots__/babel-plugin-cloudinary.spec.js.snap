// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`babel-plugin-cloudinary sample 1`] = `"const a = _transpiledURL(\\"imageName\\");"`;
